agent/config/config_windows_test.go

```diff
@@ -16,10 +16,14 @@
 package config
 
 import (
+	"errors"
 	"os"
 	"testing"
 	"time"
 
+	"github.com/hectane/go-acl/api"
```

Comment:
nit: same comment as above. Please restructure the imports here.

Reply:
and this to be like:
```
import (
	"errors"
	"os"
	"testing"
	"time"

	"golang.org/x/sys/windows"

	"github.com/aws/amazon-ecs-agent/agent/dockerclient"
	"github.com/aws/amazon-ecs-agent/agent/ec2"
	"github.com/hectane/go-acl/api"
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
)
```